apps/nextra/pages/en/build/smart-contracts/book/move-2.mdx

```diff
@@ -1,7 +1,18 @@
-# Move 2.0 Language Release
+# Move 2 Release Notes
 
-The Move 2.0 language release adds a number of new features to Move. The documentation of those features is integrated into the book, and marked in text with "_Since language version 2.0_". This section gives an overview of the new features and links to the documentation:
+The Move 2 language releases are described on this page. The reference documentation of the new features is integrated into the book, and marked in the text with "_Since language version 2.n_".
 
+## Move 2.1 
+
+The Move 2.1 language release adds the following features to Move:
+
+- **Compound Assignments** One can now use `x += n`, `x -= n`, etc. to combine assignments and arithmetic operations. See [reference doc here](variables.mdx#compound-assignments) for the supported operations.
+
+- **Loop Labels** One can now use labels for loops and have a `break` or `continue` expression refer to those labels. This allows to continue or break outer loops from within nested loops. See [reference doc here](loops.mdx#loop-labels).
```

Comment:
Assuming the relevant PR (#14962) is in the release, perhaps we should also mention the minor feature/fix: 
```
- **Underscore function parameters are wildcards, not symbols**  Function parameters named `_` no longer act like variables: they do not bind a value, and multiple such parameters to a function does not cause a conflict.  Using `_` in a value expression will yield an error, as it has no value.  This makes the behavior of `_` more like the wildcard it is in patterns and let expressions, where it does not bind a value.
```

Reply:
Too late...